cmd/kubelet/app/options/options.go

```diff
@@ -448,7 +448,7 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 	fs.StringVar(&c.ProviderID, "provider-id", c.ProviderID, "Unique identifier for identifying the node in a machine database, i.e cloudprovider")
 
 	fs.BoolVar(&c.CgroupsPerQOS, "cgroups-per-qos", c.CgroupsPerQOS, "Enable creation of QoS cgroup hierarchy, if true top level QoS and pod cgroups are created.")
-	fs.StringVar(&c.CgroupDriver, "cgroup-driver", c.CgroupDriver, "Driver that the kubelet uses to manipulate cgroups on the host.  Possible values: 'cgroupfs', 'systemd'")
+	fs.StringVar(&c.CgroupDriver, "cgroup-driver", c.CgroupDriver, "Driver that the kubelet uses to manipulate cgroups on the host.  Possible values: 'cgroupfs', 'systemd'. Note: ignored if the KubeletCgroupDriverFromCRI feature gate is enabled and a sufficiently new version of a container runtime is used, in which case the correct cgroup driver is automatically detected.")
```

Comment:
Can you be more specific about a container runtime version or re-frase this somehow? Do you mean the version that supports certain CRI API?

Reply:
Yes, this is about a version of the container runtime that supports the `RuntimeConfig` CRI rpc. In practice this means CRI-O v1.28+ and containerd 2.0+ (2.0 is not released yet).

Sure, I can reword it. I feel that talking about RuntimeConfig rpc is not useful information to the end-user. Do we want to pintpoint specific versions of container runtimes here? WDYT?

Examples:

*Note: ignored if the KubeletCgroupDriverFromCRI feature gate is enabled and a container runtime supporting RuntimeConfig rpm is used, in which case the correct cgroup driver is automatically detected.")*

OR

*Note: ignored if the KubeletCgroupDriverFromCRI feature gate is enabled and a sufficiently new version of a container runtime is used (containerd v2.0 or later, cri-o v1.28 or later), in which case the correct cgroup driver is automatically detected.")*

Reply:
Thank you. I like the former better, except `rpm` thing. I don't understand it. Is it a typo and you meant `rpc`? 
Can we use `CRI API `or something similar instead?

Reply:
Argh, my fat fingers and brain, rpm vs rpc 🙈

Changed to

*Note: ignored if the KubeletCgroupDriverFromCRI feature gate is enabled and a container runtime supporting the RuntimeConfig CRI rpc is used, in which case the correct cgroup driver is automatically detected.*

ping @sftim
you have educated opinions on these
/cc sftim

Reply:
Your last edition looks good to me. Please, update the PR and I'll lgtm it.